components/supervisor/pkg/supervisor/config.go

```diff
@@ -243,6 +243,9 @@ type WorkspaceConfig struct {
 	// WorkspaceClassInfo denotes the detail of workspace class
 	WorkspaceClassInfo *WorkspaceClassInfo `env:"GITPOD_WORKSPACE_CLASS_INFO"`
 
+	// WorkspaceDefaultImage is the default image of current workspace
+	WorkspaceDefaultImage string `env:"GITPOD_WORKSPACE_DEFAULT_IMAGE"`
```

Comment:
we should have a function here and if WorkspaceDefaultImage is emtpy then fallback to workspace-full

Reply:
Or should this settings handling/fallback a concern of server, and we just pass a `WorkspaceImage` to supervisor?

Reply:
Do you an explicit mean `getDefaultWorkspaceImage(workspaceId)` function?

Reply:
We will depend on server reliability though for `gp validate` then, but from other side it would be better for `gp validate` as well. Maybe both 😄

Reply:
> Do you mean an explicit mean getDefaultWorkspaceImage(workspaceId) API? that workspace does not need to care about org, but server can get it and figure out. It would work for me.

I'd like to have this addressed in a follow up PR, if we confirm this feedback. 🙏  @akosyakov @geropl

Reply:
fine with me, I also don't think it is exclusive, could be supervisor API which delegaetd to Public API and fllbacks to this env var if not reachable. 

If we do it now we will need split the PR.

Reply:
let's create a linear issue for it